core/src/test/java/nl/nn/adapterframework/configuration/classloaders/DatabaseClassLoaderTest.java

```diff
@@ -89,6 +92,19 @@ private void mockDatabase(boolean throwException) throws Exception {
 		doReturn(Misc.streamToBytes(file.openStream())).when(rs).getBytes(anyInt());
 		doReturn(rs).when(stmt).executeQuery();
 		doReturn(fq).when(ibisContext).createBeanAutowireByName(FixedQuerySender.class);
+
+		@SuppressWarnings("rawtypes") //IbisContext.log is a void method
+		Answer answer = new Answer() {
+			@Override
+			public Object answer(InvocationOnMock invocation) throws Throwable {
+				String message = invocation.getArgument(0);
+				MessageKeeperLevel level = invocation.getArgument(1);
+				Exception exception = invocation.getArgument(2);
+				new ApplicationMessageEvent(spy(ApplicationContext.class), message, level, exception);
+				return null;
+			}
+		};
+		doAnswer(answer).when(ibisContext).log(anyString(), any(MessageKeeperLevel.class), any(Exception.class)); //Mock the ibiscontext's log method, bypass event publishing
```

Comment:
En dit doe je om onverwachte log regels te vermijden, die een aantal tests kapot maken? Dat graag erbij in het commentaar.